nifi-nar-bundles/nifi-standard-services/nifi-hadoop-dbcp-service-bundle/nifi-hadoop-dbcp-service/src/main/java/org/apache/nifi/dbcp/HadoopDBCPConnectionPool.java

```diff
@@ -86,37 +99,13 @@
                 )
         }
 )
-public class HadoopDBCPConnectionPool extends AbstractControllerService implements DBCPService {
+public class HadoopDBCPConnectionPool extends AbstractDBCPConnectionPool {
 
     private static final String ALLOW_EXPLICIT_KEYTAB = "NIFI_ALLOW_EXPLICIT_KEYTAB";
 
     private static final String HADOOP_CONFIGURATION_CLASS = "org.apache.hadoop.conf.Configuration";
     private static final String HADOOP_UGI_CLASS = "org.apache.hadoop.security.UserGroupInformation";
 
-    private static final String DEFAULT_MIN_IDLE = "0";
-    private static final String DEFAULT_MAX_IDLE = "8";
-    private static final String DEFAULT_MAX_CONN_LIFETIME = "-1";
-    private static final String DEFAULT_EVICTION_RUN_PERIOD = String.valueOf(-1L);
-    private static final String DEFAULT_MIN_EVICTABLE_IDLE_TIME = "30 mins";
-    private static final String DEFAULT_SOFT_MIN_EVICTABLE_IDLE_TIME = String.valueOf(-1L);
-
-    public static final PropertyDescriptor DATABASE_URL = new PropertyDescriptor.Builder()
-            .name("Database Connection URL")
-            .description("A database connection URL used to connect to a database. May contain database system name, host, port, database name and some parameters."
-                    + " The exact syntax of a database connection URL is specified by your DBMS.")
-            .addValidator(StandardValidators.NON_EMPTY_VALIDATOR)
-            .required(true)
-            .expressionLanguageSupported(ExpressionLanguageScope.ENVIRONMENT)
-            .build();
-
-    public static final PropertyDescriptor DB_DRIVERNAME = new PropertyDescriptor.Builder()
-            .name("Database Driver Class Name")
-            .description("Database driver class name")
-            .required(true)
-            .addValidator(StandardValidators.NON_EMPTY_VALIDATOR)
-            .expressionLanguageSupported(ExpressionLanguageScope.ENVIRONMENT)
-            .build();
-
     public static final PropertyDescriptor DB_DRIVER_LOCATION = new PropertyDescriptor.Builder()
```

Comment:
It could be simplified via reusing and customizing the property descriptor from `DBCPProperties`:
```
        public static final PropertyDescriptor DB_DRIVER_LOCATION = new PropertyDescriptor.Builder()
            .fromPropertyDescriptor(DBCPProperties.DB_DRIVER_LOCATION)
            .description("Comma-separated list of files/folders and/or URLs containing the driver JAR and its dependencies (if any). " +
                    "For example '/var/tmp/phoenix-client.jar'. NOTE: It is required that the resources specified by this property provide " +
                    "the classes from hadoop-common, such as Configuration and UserGroupInformation.")
            .required(true)
            .build();
```

Also, the description says that some jars are required and the property is also `required(true)`. So removing "_(if any)_" would make sense.